### DISCUSSION TITLE
dryRun mode with manageable output

### DISCUSSION THREAD

**Discussion by krichter722** [renovatebot/renovate#30230] [Suggest an Idea]
### Tell us more.

I just tried the different [dryRun modes](https://docs.renovatebot.com/self-hosted-configuration/#dryrun) and noticed that none of them produces less than 4MB of logs for a very simple setup with 5 projects. This exceeds at least GitLab CI's log limit and I can imagine similar issues with other SaaS CI providers. And the information is just very verbose and most of it irrelevant. It's however great for debugging to also have a very verbose mode, so the behavior of "extract", "lookup" and "full" have their place.

It'd be nice to have a `dryRun` mode which only lists the detected projects and MRs which would be created. I'd call the mode "onboarding" or "short".

**Comment by RahulGautamSingh** [comment]
Have you tried setting `LOG_LEVEL=info`. It produces way lesser logs.

**Reply by krichter722** [reply]
Yes, that's an option. It's a bit hard to figure out how to get the information logged that I want when I have to combine all modes with `LOG_LEVEL=info`  and `LOG_LEVEL=debug`. But on the other hand that's part of the game. Thanks for your input.

**Reply by RahulGautamSingh** [reply]
You can try [logLevelRemap](https://docs.renovatebot.com/configuration-options/#loglevelremap) with `LOG_LEVEL=info` to get the debug logs you want.
